### PR TITLE
Alert banner management on javascript off 

### PR DESCRIPTION
**Pull request by akssinhaamit**
[Trello](https://trello.com/c/y1rYRJQb/209-s29-alert-banner-management-javascript-off)

### CODE REVIEW

File: response_operations_ui/templates/admin/banner-manage.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 {% from "components/radios/_macro.njk" import onsRadios %}
 {% set page_title = 'Alert Admin'  %}
 {% block main %}
-  <div class="grid grid--gutterless">
+<div class="grid grid--gutterless" xmlns="http://www.w3.org/1999/html">
```
Comment (akssinhaamit) [inline response_operations_ui/templates/admin/banner-manage.html:10]: this has been added in mistake will take it out.

**Review by mfcrocker — APPROVED** [review]
LGTM

**Pull request merged**